packages/harmony/src/components/typography/index.ts

```diff
@@ -1 +1,2 @@
+export * from './Icons'
```

Comment:
Confirming that we want icons as part of typography

Reply:
yea made sense to me